buildspecs/release-javadoc.yml

```diff
@@ -23,6 +23,7 @@ phases:
     - MODULES_TO_SKIP_FORMATTED=$(echo $MODULES_TO_SKIP | sed 's/,/,!:/g' | sed 's/^/!:/')
     - mvn clean install javadoc:aggregate -B -Ppublic-javadoc -Dcheckstyle.skip -Dspotbugs.skip -DskipTests -Ddoclint=none -pl $MODULES_TO_SKIP_FORMATTED
     - RELEASE_VERSION=`mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec`
+    - python3 ./scripts/code_examples/inject_examples.py --exampleMetadata ./codegen/src/main/resources/software/amazon/awssdk/codegen/example-meta.json --docsPath ./target/site/apidocs/
```

Comment:
Have we considered updating codegen code to link to the examples, similar to https://github.com/aws/aws-sdk-java-v2/pull/6626

https://github.com/aws/aws-sdk-java-v2/blob/master/codegen/src/main/java/software/amazon/awssdk/codegen/emitters/tasks/PackageInfoGeneratorTasks.java#L29